cmake/Modules/FindCoreFoundation.cmake

```diff
@@ -1,9 +1,30 @@
-IF (COREFOUNDATION_INCLUDE_DIR AND COREFOUNDATION_DIRS)
-  SET(COREFOUNDATION_FOUND TRUE)
+# Find CoreFoundation.framework
+# This will define :
+#
+# COREFOUNDATION_FOUND
+# COREFOUNDATION_LIBRARIES
+# COREFOUNDATION_LDFLAGS
+#
+
+IF (COREFOUNDATION_INCLUDE_DIR AND COREFOUNDATION_LIBRARIES)
+	SET(COREFOUNDATION_FOUND TRUE)
 ELSE ()
-  FIND_PATH(COREFOUNDATION_INCLUDE_DIR NAMES CoreFoundation.h)
-  FIND_LIBRARY(COREFOUNDATION_DIRS NAMES CoreFoundation)
-  IF (COREFOUNDATION_INCLUDE_DIR AND COREFOUNDATION_DIRS)
-    SET(COREFOUNDATION_FOUND TRUE)
-  ENDIF ()
+	FIND_PATH(COREFOUNDATION_INCLUDE_DIR NAMES CoreFoundation.h)
+	FIND_LIBRARY(COREFOUNDATION_LIBRARIES NAMES CoreFoundation)
+	IF (COREFOUNDATION_INCLUDE_DIR AND COREFOUNDATION_LIBRARIES)
+		IF (NOT CoreFoundation_FIND_QUIETLY)
+			MESSAGE("-- Found CoreFoundation ${COREFOUNDATION_LIBRARIES}")
+		ENDIF()
+		SET(COREFOUNDATION_FOUND TRUE)
+		SET(COREFOUNDATION_LDFLAGS "-framework CoreFoundation")
```

Comment:
The LDFLAGS variable may not be set in the first case, where you only check whether COREFOUNDATION_INCLUDE_DIR and COREFOUNDATION_LIBRARIES are set. If you touch this place again, I'd love if you could split this up into two patches: the first one fixes indentation only and the second one introduces your changes. This would make it a bit easier to review.

Reply:
I'll add a reindent commit before then.

About those changes, what I'm attempting to do is stopping the `cmake` run from echoing

```
-- Found Security /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/System/Library/Frameworks/Security.framework
-- Found CoreFoundation /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/System/Library/Frameworks/CoreFoundation.framework
```

on every run. Since after the first run, everything gets cached, so this just short-circuit the test. I can also make both `FIND_PACKAGE(... QUIET REQUIRED)`, but this silences it completely...

Reply:
I ended up removing my bad attempt at using the cache correctly.